src/test/java/dasniko/testcontainers/keycloak/KeycloakContainerTest.java

```diff
@@ -32,6 +34,7 @@
 public class KeycloakContainerTest {
 
     public static final String TEST_REALM_JSON = "/test-realm.json";
+    public static final String MASTER_REALM_USERS_JSON = "/master-realm.json";
```

Comment:
Just something minor: "master-realm-with-admin-user" would be a better naming fit, IMHO. Both, for constant and file name.